test/plugin.test.js

```diff
@@ -96,6 +96,7 @@ QUnit.test('hls lifecyle', function(assert) {
 
 QUnit.test('dash lifecyle', function(assert) {
   let mediaPlayer = new MockMediaPlayer();
+  const oldBeforeInitialize = videojs.Html5DashJS.beforeInitialize;
```

Comment:
No I believe it is just undefined, but probably a good idea to set it to some function to make sure it restored the function correctly